Assets/Scripts/Simulation/SimulationBase.cs

```diff
@@ -121,6 +121,11 @@ public virtual void SetScenario(TScenario scenario)
             CommunicationManager.SetRobotRelativeSize(scenario.RobotConstraints.AgentRelativeSize);
             foreach (var robot in Robots)
             {
+                if (scenario is PatrollingSimulationScenario simulationScenario)
+                {
+                    // Assign robots to partitions, distributed evenly.
+                    robot.AssignedPartition = (((robot.id + 1) % simulationScenario.Partitions) + 1);
```

Comment:
Why are we not doing just robot.id % simulationScenario.Partitions?

Reply:
Long story, partitions are not 0 indexed. Will be fixed in a future PR.

Reply:
Ok but the code here is buggy no?
Shouldnt it be: `(robot.id % simulationScenario.Partitions) + 1